ui/apps/platform/src/Containers/Violations/violationTableColumnDescriptors.tsx

```diff
@@ -100,9 +101,20 @@ const tableColumnDescriptor = [
         Cell: EntityTableCell,
     },
     {
-        Header: 'Type',
+        Header: 'Kind',
         accessor: 'commonEntityInfo.resourceType',
-        Cell: ({ value }): string => startCase(value.toLowerCase()),
+        Cell: ({ value, original }): string => {
+            const deployment = original?.deployment || {};
+            if (
+                value === resourceTypes.DEPLOYMENT &&
+                deployment.deploymentType &&
+                typeof deployment.deploymentType === 'string' &&
+                deployment.deploymentType.length > 0
+            ) {
+                return deployment.deploymentType as string;
+            }
+            return startCase(value.toLowerCase());
+        },
```

Comment:
My only concern is that we display the `Deployment type` or the `Resource type` here, but we display `Kind` as the table column header. If the type is not a `Deployment`, then it might be confusing. We will need to figure out a good word to use here. (cc: @alanonthegit )